apps/mivisionx_winml_yolov2/README.md

```diff
@@ -0,0 +1,50 @@
+# YOLOv2 using AMD winml (AMD NN Inference Engine)
+
+*This project shows how to run tiny yolov2 (20 classes) with AMD's NN inference engine(Annie):*
++ A c/c++ implementation for region layer of yolov2
++ A sample for running yolov2 with Annie
+
+---
+
+### Preliminaries
```

Comment:
# MIVisionX_WinML-installer.msi - Prerequisites

* Windows 10, version 1809 or later
* Windows SDK, build 17763 or later
* Visual Studio 2017, version 15.7.4 or later
* Visual Studio extension for C++/WinRT
* Install OpenCL SDK
* OpenCV 3.4
    * Set OpenCV_DIR environment variable to OpenCV/build folder
    * Add %OpenCV_DIR%\x64\vc14\bin or %OpenCV_DIR%\x64\vc15\bin to your PATH